modules/gitgraph/graph.go

```diff
@@ -24,35 +24,30 @@ func GetCommitGraph(r *git.Repository, page, maxAllowedColors int, hidePRRefs bo
 		page = 1
 	}
 
-	args := make([]string, 0, 12+len(branches)+len(files))
-
-	args = append(args, "--graph", "--date-order", "--decorate=full")
+	graphCmd := git.NewCommand(r.Ctx, "log", "--graph", "--date-order", "--decorate=full")
 
 	if hidePRRefs {
-		args = append(args, "--exclude="+git.PullPrefix+"*")
+		graphCmd.AddArguments("--exclude=" + git.PullPrefix + "*")
```

Comment:
… We certainly have to agree at some point whether we want spaces around string concatenation, or not.
Some maintainers (like me) prefer spaces, and others don't want them. I've seen such changes now too often…

Reply:
That's the correct Golang format.


![image](https://user-images.githubusercontent.com/2114189/195986093-5c0aa82f-4b34-4bdf-ac31-7aacaa9667ef.png)

Reply:
Golang has different styles for `append()` or and function call.

Reply:
yes ... and in this case as long as the ci dont complain we should leave it as is